auth0/resource_auth0_user_test.go

```diff
@@ -53,7 +53,6 @@ provider "auth0" {}
 
 resource "auth0_user" "user" {
   connection_name = "Username-Password-Authentication"
-  username = "test"
```

Comment:
This will need to be set so it can pass during CI. We’ve enabled “requires username” on this connection a while ago and we require it everywhere.

Reply:
Thanks @alexkappa  - I've added the `username` back in. Any ideas on how to solve the next CI error `"client_secret" is required, but no definition was found.`?

Reply:
That is an unfortunate side effect of switching to travis. It seems that PRs don't get access to all secrets for security reasons. I'll need to run it manually I suppose.